gradle.properties

```diff
@@ -0,0 +1,2 @@
+
+org.gradle.jvmargs=-XX:UseAVX=0
```

Comment:
This is not a requirement of the project - please revert.
(You can configure such JVM options locally as necessary.)

Reply:
please remove this.

Reply:
Added comments to justify the JVM setting

Reply:
Thanks